cli/package.json

```
@@ -2,15 +2,15 @@
   "name": "@iexec/iapp",
   "version": "0.0.1-alpha",
   "description": "A CLI to guide you through the process of building an iExec iApp",
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "type": "module",
   "bin": {
-    "iapp": "src/index.js"
+    "iapp": "dist/index.js"
   },
   "homepage": "https://github.com/iExecBlockchainComputing/iapp/blob/main/cli/README.md",
   "scripts": {
-    "start": "./src/index.js --help",
-    "iapp": "./src/index.js",
+    "build": "rimraf dist && tsc --project tsconfig.json",
+    "build:watch": "npm run build -- --watch",
     "check-format": "prettier --check .",
     "format": "prettier --write .",
     "lint": "eslint src"
```

```suggestion
    "lint": "eslint src",
    "check-types": "tsc --project tsconfig.json"
```

then add step in drone CI 

  - name: check-types
    image: node:20
    commands:
      - cd cli
      - npm run check-types
    depends_on:
      - install-deps

---

build already does the type-checking